assignments/callbacks.js

```diff
@@ -27,25 +27,54 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, (lengthofArray) => {
+  console.log(lengthofArray);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length - 1]);
 }
 
+last(items, (lastItem) => {
+  console.log(lastItem);
+});
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
+sumNums(2, 2, (sum) => {
+  console.log(sum);
+});
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
 
+multiplyNums(2, 2, (product) => {
+  console.log(product);
+})
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  cb(list.includes(item));
 }
 
+contains("potato", items, (statement) => {
+  console.log(statement);
+})
+
+contains("Gum", items, (statement) => {
+  console.log(statement);
+})
```

Comment:
good work so far, instead of re writing the same cb over and over, for the sake of keeping things DRY, one could write a single func like this `const logger = value => console.log(value)` and pass it in where needed